optuna/trial.py

```diff
@@ -283,13 +320,29 @@ def suggest_int(self, name, low, high):
             Suggest the number of trees in `RandomForestClassifier <https://scikit-learn.org/
             stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html>`_.
 
-            .. code::
+            .. testcode::
+
+                import optuna
+                import numpy as np
+                from sklearn.ensemble import RandomForestClassifier
+                from sklearn.model_selection import train_test_split
+
+                def objective(trial):
+                    n_estimators = trial.suggest_int('n_estimators', 50, 400)
+                    assert 50 <= n_estimators <= 400
+
+                    np.random.seed(seed=0)
+                    X = np.random.randn(50).reshape(-1, 1)
+                    y = np.random.randint(0, 2, 50)
+                    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
```

Comment:
Please hide this dataset preparation, as I suggested in the `trial.suggest_loguniform` example.